src/server/server.cc

```diff
@@ -131,12 +131,16 @@ Server::~Server() {
 //     - feed-replica-data-info: generate checkpoint and send files list when full sync
 //     - feed-replica-file: send SST files when slaves ask for full sync
 Status Server::Start() {
+  auto s = namespace_.Load();
```

Comment:
It is loaded despite the value of `repl-namespace-enabled`?

Reply:
Yes, it will only load tokens from the configuration file if the repl-namespace-enabled is disabled.

Reply:
https://github.com/apache/kvrocks/pull/1776/files#diff-e7439881acdda2d7b193156d6255f5691a76bd6b1bc69ddd46432a6c3903ad68R54 Here we load tokens from storage despite this value?

Reply:
Yes, we still need to load tokens from DB even though it is disabled, because it may lose the db token if users switch from yes to no when restarting the server.